libvast/src/table_slice.cpp

```diff
@@ -625,22 +625,6 @@ struct row_evaluator {
       }
       return neg ? !result : result;
     }
-    if (e.attr == atom::timestamp_v) {
-      for (size_t col = 0; col < layout.fields.size(); ++col) {
-        auto& field = layout.fields[col];
-        if (has_attribute(field.type, "timestamp")) {
-          if (!caf::holds_alternative<time_type>(field.type)) {
-            VAST_WARN("got timestamp attribute for non-time type");
-            return false;
-          }
-        }
-        auto lhs = to_canonical(
```

Comment:
Is `to_canonical` still needed?

Reply:
`to_canonical` is only needed for enumerations at the moment, the function is still used in other places.